boards/posix/native_posix/doc/board.rst

```diff
@@ -519,6 +519,15 @@ The following peripherals are currently provided with this board:
   must be powered down and support Bluetooth Low Energy (i.e. support the
   Bluetooth specification version 4.0 or greater).
 
+**Display driver**:
+  A display driver is provided that will create a window on the host machine to
+  render the display content. Underlying the display driver is using the
+  `SDL2`_ library for rendering and as such requires a 32-bit version of the
+  SDL2 library installed on the host machine. Further there is a dependency on
+  pkg-config to correctly pickup the SDL2 install path and compiler flags.
```

Comment:
I feel this description alone is going to be too little for quite a few users. (Even when we give exact steps with packages to install some people has issues..)
In Ubuntu 18.04 I needed to install `pkg-config` and `libsdl2-dev:i386`, but also I needed to `export PKG_CONFIG_PATH=/usr/lib/i386-linux-gnu/pkgconfig/` before calling cmake. This is not too trivial and probably will cause a bug report as soon as somebody tries it.

Reply:
Added a section on which packages to install and the export